utilities/DumpUnitaryTest/DumpUnitaryTest.csproj

```diff
@@ -8,7 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="xunit" Version="2.3.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" />
```

Comment:
In all the other files this got bumped to 2.4.5. Why only to 2.4.1 here?

Reply:
To work around the confusing issue I was asking about in the Teams.